### PR TITLE
Return nil directly if err is nil and maintain code style consistency

### PR DESCRIPTION
**Pull request by zhaosijun**
I found the code style is different.  I think maintaining code style consistency is more favorable

### CODE REVIEW

File: pkg/volume/emptydir/empty_dir_quota.go [hunk 1]
```diff
@@ -69,10 +69,16 @@ func (edq *emptyDirQuotaMounter) SetUp(fsGroup *int64) error {
...
+		return err
 	}
-	return err
+
+	err = edq.quotaApplicator.Apply(dir, edq.medium, edq.pod, fsGroup, edq.quota)
+	if err != nil {
+		return err
+	}
+
+	return nil
```
Comment (dgoodwin) [inline pkg/volume/emptydir/empty_dir_quota.go]: Should we just return "err" here to be a little more concise and drop the "if" above? 

Could you link to the style consistency you refer to here for my own knowledge?

Comment (zhaosijun) [inline pkg/volume/emptydir/empty_dir_quota.go]: I aggre with you that just return "err" here to be a little more concise. But When I read code, I foud most code like below , Mabye we would like to add logs in the "if", or develop the fuction in future.
pkg/volume/emptydir/quota.go
	if fsGroup == nil {
		// This indicates the operation matched an SCC with FSGroup strategy RunAsAny.
		// Not an error condition.
		glog.V(5).Infof("Unable to apply XFS quota, no FSGroup specified.")
		return nil
	}

	volDevice, err := xqa.getFSDevice(dir)
	if err != nil {
		return err
	}

	err = xqa.applyQuota(volDevice, quota, *fsGroup)
	if err != nil {
		return err
	}

	return nil
}

Comment (dgoodwin) [inline pkg/volume/emptydir/empty_dir_quota.go]: Ok that's fine with me, patch looks good to merge IMO.

**Comment by pweil-** [comment]
closing due to inactivity

**Pull request closed without merging**